dlrover/python/tests/test_ray_job_args.py

```diff
@@ -25,6 +25,9 @@
 
 
 class RayJobArgsTest(unittest.TestCase):
+    def tearDown(self):
+        os.remove("test.json")
+
     def test_initialize_params(self):
         file = "data/demo.yaml"
```

Comment:
it would be better to use `tempfile` to manage temporary file.

Reply:
Can you fix it later? I don't know what is the usage of `test.json` in the UT.